rules/ember.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  plugins: [
+    'ember',
+  ],
+  rules: {
+    // General
+    'ember/local-modules': 'off',
+    'ember/no-observers': 'off',
+    'ember/no-side-effect': 'off',
+    'ember/jquery-ember-run': 'off',
```

Comment:
I'm not too confident about the implementation of that rule and whether that will lead to many false positives. I'd propose to try this rule out in a few projects and see if it works properly before enabling it here in the main config.